# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build gstreamer-build libvorbis-build orc-build pango-build"

PKG_CONFIG_PATH=/drm/share/pkgconfig pkg-config --exists --print-errors $REQUIRES || exit 1

if pkg-config --exists alsa-lib-build; then
  ALSA=1
  REQUIRES="$REQUIRES alsa-lib-build"
fi

if PKG_CONFIG_PATH=/drm/share/pkgconfig pkg-config --exists egl-opengl-stubs-build gudev-build libdrm-build libgbm-stub-build; then
  GL=1
  REQUIRES="$REQUIRES egl-opengl-stubs-build gudev-build libdrm-build libgbm-stub-build"
fi

if pkg-config --exists opus-build; then
  OPUS=1
  REQUIRES="$REQUIRES opus-build"
fi

if pkg-config --exists tremor-build; then
  TREMOR=1
  REQUIRES="$REQUIRES tremor-build"
fi

# configure (ac-2.69; am-1.16; lt-2.4.6)
libtoolize -c -f; aclocal -I m4 -I common/m4; autoheader; autoconf; automake -a -c
PKG_CONFIG_PATH=/drm/lib/pkgconfig LDFLAGS=-Wl,-rpath,/drm/lib ./configure --prefix=/drm

# build
make

# install
install -d $DESTDIR/drm/bin
install tools/.libs/gst-device-monitor-1.0 $DESTDIR/drm/bin
install tools/.libs/gst-discoverer-1.0 $DESTDIR/drm/bin
install tools/.libs/gst-play-1.0 $DESTDIR/drm/bin
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/allocators
install -m 644 gst-libs/gst/allocators/allocators.h $DESTDIR/drm/include/gstreamer-1.0/gst/allocators
install -m 644 gst-libs/gst/allocators/allocators-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/allocators
install -m 644 gst-libs/gst/allocators/gstdmabuf.h $DESTDIR/drm/include/gstreamer-1.0/gst/allocators
install -m 644 gst-libs/gst/allocators/gstfdmemory.h $DESTDIR/drm/include/gstreamer-1.0/gst/allocators
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/app
install -m 644 gst-libs/gst/app/app.h $DESTDIR/drm/include/gstreamer-1.0/gst/app
install -m 644 gst-libs/gst/app/app-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/app
install -m 644 gst-libs/gst/app/app-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/app
install -m 644 gst-libs/gst/app/gstappsink.h $DESTDIR/drm/include/gstreamer-1.0/gst/app
install -m 644 gst-libs/gst/app/gstappsrc.h $DESTDIR/drm/include/gstreamer-1.0/gst/app
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-buffer.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-channel-mixer.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-channels.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-converter.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-format.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-info.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-quantize.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/audio-resampler.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudioaggregator.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiobasesink.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiobasesrc.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiocdsrc.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudioclock.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiodecoder.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudioencoder.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiofilter.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudioiec61937.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiometa.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudioringbuffer.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiosink.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiosrc.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/gstaudiostreamalign.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -m 644 gst-libs/gst/audio/streamvolume.h $DESTDIR/drm/include/gstreamer-1.0/gst/audio
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/fft
install -m 644 gst-libs/gst/fft/fft.h $DESTDIR/drm/include/gstreamer-1.0/gst/fft
install -m 644 gst-libs/gst/fft/fft-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/fft
install -m 644 gst-libs/gst/fft/gstfft.h $DESTDIR/drm/include/gstreamer-1.0/gst/fft
install -m 644 gst-libs/gst/fft/gstfftf32.h $DESTDIR/drm/include/gstreamer-1.0/gst/fft
install -m 644 gst-libs/gst/fft/gstfftf64.h $DESTDIR/drm/include/gstreamer-1.0/gst/fft
install -m 644 gst-libs/gst/fft/gstffts16.h $DESTDIR/drm/include/gstreamer-1.0/gst/fft
install -m 644 gst-libs/gst/fft/gstffts32.h $DESTDIR/drm/include/gstreamer-1.0/gst/fft
test $GL && install -d $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gl.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gl-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gl-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstgl_enums.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstgl_fwd.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglapi.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglbasefilter.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglbasememory.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglbuffer.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglbufferpool.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglcolorconvert.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglconfig.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglcontext.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstgldebug.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstgldisplay.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglfeature.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglfilter.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglformat.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglframebuffer.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglfuncs.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglmemory.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglmemorypbo.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstgloverlaycompositor.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglquery.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglrenderbuffer.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglshader.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglshaderstrings.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglsl.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglslstage.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglsyncmeta.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglupload.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglutils.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglviewconvert.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -m 644 gst-libs/gst/gl/gstglwindow.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl
test $GL && install -d $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/all_functions.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/base.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/blending.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/buffers.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/debug.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/eglimage.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/fbo.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/fixedfunction.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/gles.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/gstgl_compat.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/gstgl_gles2compat.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/opengl.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/query.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/shaders.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/sync.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
test $GL && install -m 644 gst-libs/gst/gl/glprototypes/vao.h $DESTDIR/drm/include/gstreamer-1.0/gst/gl/glprototypes
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/codec-utils.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/descriptions.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/encoding-profile.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/encoding-target.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/gstaudiovisualizer.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/gstdiscoverer.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/gstpluginsbaseversion.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/install-plugins.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/missing-plugins.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/pbutils.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/pbutils-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -m 644 gst-libs/gst/pbutils/pbutils-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/pbutils
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/riff
install -m 644 gst-libs/gst/riff/riff.h $DESTDIR/drm/include/gstreamer-1.0/gst/riff
install -m 644 gst-libs/gst/riff/riff-ids.h $DESTDIR/drm/include/gstreamer-1.0/gst/riff
install -m 644 gst-libs/gst/riff/riff-media.h $DESTDIR/drm/include/gstreamer-1.0/gst/riff
install -m 644 gst-libs/gst/riff/riff-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/riff
install -m 644 gst-libs/gst/riff/riff-read.h $DESTDIR/drm/include/gstreamer-1.0/gst/riff
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtcpbuffer.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtp-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtpbaseaudiopayload.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtpbasedepayload.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtpbasepayload.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtpbuffer.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtpdefs.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtphdrext.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtpmeta.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/gstrtppayloads.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/rtp.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -m 644 gst-libs/gst/rtp/rtp-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtp
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtsp-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtspconnection.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtspdefs.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtspextension.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtspmessage.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtsprange.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtsptransport.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/gstrtspurl.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/rtsp.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -m 644 gst-libs/gst/rtsp/rtsp-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/rtsp
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/sdp
install -m 644 gst-libs/gst/sdp/gstmikey.h $DESTDIR/drm/include/gstreamer-1.0/gst/sdp
install -m 644 gst-libs/gst/sdp/gstsdp.h $DESTDIR/drm/include/gstreamer-1.0/gst/sdp
install -m 644 gst-libs/gst/sdp/gstsdpmessage.h $DESTDIR/drm/include/gstreamer-1.0/gst/sdp
install -m 644 gst-libs/gst/sdp/sdp.h $DESTDIR/drm/include/gstreamer-1.0/gst/sdp
install -m 644 gst-libs/gst/sdp/sdp-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/sdp
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/tag
install -m 644 gst-libs/gst/tag/gsttagdemux.h $DESTDIR/drm/include/gstreamer-1.0/gst/tag
install -m 644 gst-libs/gst/tag/gsttagmux.h $DESTDIR/drm/include/gstreamer-1.0/gst/tag
install -m 644 gst-libs/gst/tag/tag.h $DESTDIR/drm/include/gstreamer-1.0/gst/tag
install -m 644 gst-libs/gst/tag/tag-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/tag
install -m 644 gst-libs/gst/tag/tag-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/tag
install -m 644 gst-libs/gst/tag/xmpwriter.h $DESTDIR/drm/include/gstreamer-1.0/gst/tag
install -d $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/colorbalance.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/colorbalancechannel.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideoaffinetransformationmeta.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideoaggregator.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideodecoder.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideoencoder.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideofilter.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideometa.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideopool.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideosink.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideotimecode.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/gstvideoutils.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/navigation.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-anc.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-blend.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-chroma.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-color.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-converter.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-dither.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-enumtypes.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-event.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-format.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-frame.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-info.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-multiview.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-overlay-composition.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-prelude.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-resampler.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-scaler.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/video-tile.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/videodirection.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/videoorientation.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -m 644 gst-libs/gst/video/videooverlay.h $DESTDIR/drm/include/gstreamer-1.0/gst/video
install -d $DESTDIR/drm/lib
install gst-libs/gst/allocators/.libs/libgstallocators-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstallocators-1.0.so.0 $DESTDIR/drm/lib/libgstallocators-1.0.so
install gst-libs/gst/app/.libs/libgstapp-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstapp-1.0.so.0 $DESTDIR/drm/lib/libgstapp-1.0.so
install gst-libs/gst/audio/.libs/libgstaudio-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstaudio-1.0.so.0 $DESTDIR/drm/lib/libgstaudio-1.0.so
install gst-libs/gst/fft/.libs/libgstfft-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstfft-1.0.so.0 $DESTDIR/drm/lib/libgstfft-1.0.so
test $GL && install gst-libs/gst/gl/.libs/libgstgl-1.0.so.0 $DESTDIR/drm/lib
test $GL && ln -sf libgstgl-1.0.so.0 $DESTDIR/drm/lib/libgstgl-1.0.so
install gst-libs/gst/pbutils/.libs/libgstpbutils-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstpbutils-1.0.so.0 $DESTDIR/drm/lib/libgstpbutils-1.0.so
install gst-libs/gst/riff/.libs/libgstriff-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstriff-1.0.so.0 $DESTDIR/drm/lib/libgstriff-1.0.so
install gst-libs/gst/rtp/.libs/libgstrtp-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstrtp-1.0.so.0 $DESTDIR/drm/lib/libgstrtp-1.0.so
install gst-libs/gst/rtsp/.libs/libgstrtsp-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstrtsp-1.0.so.0 $DESTDIR/drm/lib/libgstrtsp-1.0.so
install gst-libs/gst/sdp/.libs/libgstsdp-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstsdp-1.0.so.0 $DESTDIR/drm/lib/libgstsdp-1.0.so
install gst-libs/gst/tag/.libs/libgsttag-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgsttag-1.0.so.0 $DESTDIR/drm/lib/libgsttag-1.0.so
install gst-libs/gst/video/.libs/libgstvideo-1.0.so.0 $DESTDIR/drm/lib
ln -sf libgstvideo-1.0.so.0 $DESTDIR/drm/lib/libgstvideo-1.0.so
install -d $DESTDIR/drm/lib/gstreamer-1.0
test $ALSA && install ext/alsa/.libs/libgstalsa.so $DESTDIR/drm/lib/gstreamer-1.0
test $GL && install ext/gl/.libs/libgstopengl.so $DESTDIR/drm/lib/gstreamer-1.0
install ext/ogg/.libs/libgstogg.so $DESTDIR/drm/lib/gstreamer-1.0
test $OPUS && install ext/opus/.libs/libgstopus.so $DESTDIR/drm/lib/gstreamer-1.0
install ext/pango/.libs/libgstpango.so $DESTDIR/drm/lib/gstreamer-1.0
test $TREMOR && install ext/vorbis/.libs/libgstivorbisdec.so $DESTDIR/drm/lib/gstreamer-1.0
install ext/vorbis/.libs/libgstvorbis.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/adder/.libs/libgstadder.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/app/.libs/libgstapp.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/audioconvert/.libs/libgstaudioconvert.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/audiomixer/.libs/libgstaudiomixer.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/audiorate/.libs/libgstaudiorate.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/audioresample/.libs/libgstaudioresample.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/audiotestsrc/.libs/libgstaudiotestsrc.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/compositor/.libs/libgstcompositor.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/encoding/.libs/libgstencoding.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/gio/.libs/libgstgio.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/overlaycomposition/.libs/libgstoverlaycomposition.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/pbtypes/.libs/libgstpbtypes.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/playback/.libs/libgstplayback.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/rawparse/.libs/libgstrawparse.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/subparse/.libs/libgstsubparse.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/tcp/.libs/libgsttcp.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/typefind/.libs/libgsttypefindfunctions.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/videoconvert/.libs/libgstvideoconvert.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/videorate/.libs/libgstvideorate.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/videoscale/.libs/libgstvideoscale.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/videotestsrc/.libs/libgstvideotestsrc.so $DESTDIR/drm/lib/gstreamer-1.0
install gst/volume/.libs/libgstvolume.so $DESTDIR/drm/lib/gstreamer-1.0
install -d $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-allocators-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-app-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-audio-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-fft-1.0.pc $DESTDIR/drm/lib/pkgconfig
test $GL && install -m 644 pkgconfig/gstreamer-gl-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-pbutils-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-plugins-base-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-riff-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-rtp-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-rtsp-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-sdp-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-tag-1.0.pc $DESTDIR/drm/lib/pkgconfig
install -m 644 pkgconfig/gstreamer-video-1.0.pc $DESTDIR/drm/lib/pkgconfig

# build.pc
install -d $DESTDIR/drm/share/pkgconfig
cat > $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
Name: gst-plugins-base
Version: 1.16.3
Description: Base GStreamer plugins and libraries
Requires: $REQUIRES

devel=\\
/drm/include/gstreamer-1.0/gst/allocators/allocators.h \\
/drm/include/gstreamer-1.0/gst/allocators/allocators-prelude.h \\
/drm/include/gstreamer-1.0/gst/allocators/gstdmabuf.h \\
/drm/include/gstreamer-1.0/gst/allocators/gstfdmemory.h \\
/drm/include/gstreamer-1.0/gst/app/app.h \\
/drm/include/gstreamer-1.0/gst/app/app-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/app/app-prelude.h \\
/drm/include/gstreamer-1.0/gst/app/gstappsink.h \\
/drm/include/gstreamer-1.0/gst/app/gstappsrc.h \\
/drm/include/gstreamer-1.0/gst/audio/audio.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-buffer.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-channel-mixer.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-channels.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-converter.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-format.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-info.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-prelude.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-quantize.h \\
/drm/include/gstreamer-1.0/gst/audio/audio-resampler.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudioaggregator.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiobasesink.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiobasesrc.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiocdsrc.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudioclock.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiodecoder.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudioencoder.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiofilter.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudioiec61937.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiometa.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudioringbuffer.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiosink.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiosrc.h \\
/drm/include/gstreamer-1.0/gst/audio/gstaudiostreamalign.h \\
/drm/include/gstreamer-1.0/gst/audio/streamvolume.h \\
/drm/include/gstreamer-1.0/gst/fft/fft.h \\
/drm/include/gstreamer-1.0/gst/fft/fft-prelude \\
/drm/include/gstreamer-1.0/gst/fft/gstfft.h \\
/drm/include/gstreamer-1.0/gst/fft/gstfftf32.h \\
/drm/include/gstreamer-1.0/gst/fft/gstfftf64.h \\
/drm/include/gstreamer-1.0/gst/fft/gstffts16.h \\
/drm/include/gstreamer-1.0/gst/fft/gstffts32.h \\
EOF
test $GL && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/include/gstreamer-1.0/gst/gl/gl.h \\
/drm/include/gstreamer-1.0/gst/gl/gl-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/gl/gl-prelude.h \\
/drm/include/gstreamer-1.0/gst/gl/gstgl_enums.h \\
/drm/include/gstreamer-1.0/gst/gl/gstgl_fwd.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglapi.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglbasefilter.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglbasememory.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglbuffer.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglbufferpool.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglcolorconvert.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglconfig.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglcontext.h \\
/drm/include/gstreamer-1.0/gst/gl/gstgldebug.h \\
/drm/include/gstreamer-1.0/gst/gl/gstgldisplay.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglfeature.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglfilter.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglformat.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglframebuffer.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglfuncs.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglmemory.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglmemorypbo.h \\
/drm/include/gstreamer-1.0/gst/gl/gstgloverlaycompositor.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglquery.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglrenderbuffer.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglshader.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglshaderstrings.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglsl.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglslstage.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglsyncmeta.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglupload.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglutils.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglviewconvert.h \\
/drm/include/gstreamer-1.0/gst/gl/gstglwindow.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/all_functions.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/base.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/blending.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/buffers.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/debug.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/eglimage.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/fbo.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/fixedfunction.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/gles.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/gstgl_compat.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/gstgl_gles2compat.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/opengl.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/query.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/shaders.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/sync.h \\
/drm/include/gstreamer-1.0/gst/gl/glprototypes/vao.h \\
EOF
cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/include/gstreamer-1.0/gst/pbutils/codec-utils.h \\
/drm/include/gstreamer-1.0/gst/pbutils/descriptions.h \\
/drm/include/gstreamer-1.0/gst/pbutils/encoding-profile.h \\
/drm/include/gstreamer-1.0/gst/pbutils/encoding-target.h \\
/drm/include/gstreamer-1.0/gst/pbutils/gstaudiovisualizer.h \\
/drm/include/gstreamer-1.0/gst/pbutils/gstdiscoverer.h \\
/drm/include/gstreamer-1.0/gst/pbutils/gstpluginsbaseversion.h \\
/drm/include/gstreamer-1.0/gst/pbutils/install-plugins.h \\
/drm/include/gstreamer-1.0/gst/pbutils/missing-plugins.h \\
/drm/include/gstreamer-1.0/gst/pbutils/pbutils.h \\
/drm/include/gstreamer-1.0/gst/pbutils/pbutils-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/pbutils/pbutils-prelude.h \\
/drm/include/gstreamer-1.0/gst/riff/riff.h \\
/drm/include/gstreamer-1.0/gst/riff/riff-ids.h \\
/drm/include/gstreamer-1.0/gst/riff/riff-media.h \\
/drm/include/gstreamer-1.0/gst/riff/riff-prelude.h \\
/drm/include/gstreamer-1.0/gst/riff/riff-read.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtcpbuffer.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtp-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtpbaseaudiopayload.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtpbasedepayload.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtpbasepayload.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtpbuffer.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtpdefs.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtphdrext.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtpmeta.h.h \\
/drm/include/gstreamer-1.0/gst/rtp/gstrtppayloads.h \\
/drm/include/gstreamer-1.0/gst/rtp/rtp-prelude.h \\
/drm/include/gstreamer-1.0/gst/rtp/rtp.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtsp-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtspconnection.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtspdefs.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtspextension.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtspmessage.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtsprange.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtsptransport.h \\
/drm/include/gstreamer-1.0/gst/rtsp/gstrtspurl.h \\
/drm/include/gstreamer-1.0/gst/rtsp/rtsp.h \\
/drm/include/gstreamer-1.0/gst/rtsp/rtsp-prelude.h \\
/drm/include/gstreamer-1.0/gst/sdp/gstmikey.h \\
/drm/include/gstreamer-1.0/gst/sdp/gstsdp.h \\
/drm/include/gstreamer-1.0/gst/sdp/gstsdpmessage.h \\
/drm/include/gstreamer-1.0/gst/sdp/sdp.h \\
/drm/include/gstreamer-1.0/gst/sdp/sdp-prelude.h \\
/drm/include/gstreamer-1.0/gst/tag/gsttagdemux.h \\
/drm/include/gstreamer-1.0/gst/tag/gsttagmux.h \\
/drm/include/gstreamer-1.0/gst/tag/tag.h \\
/drm/include/gstreamer-1.0/gst/tag/tag-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/tag/tag-prelude.h \\
/drm/include/gstreamer-1.0/gst/tag/xmpwriter.h \\
/drm/include/gstreamer-1.0/gst/video/colorbalance.h \\
/drm/include/gstreamer-1.0/gst/video/colorbalancechannel.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideoaffinetransformationmeta.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideoaggregator.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideodecoder.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideoencoder.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideofilter.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideometa.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideopool.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideosink.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideotimecode.h \\
/drm/include/gstreamer-1.0/gst/video/gstvideoutils.h \\
/drm/include/gstreamer-1.0/gst/video/navigation.h \\
/drm/include/gstreamer-1.0/gst/video/video.h \\
/drm/include/gstreamer-1.0/gst/video/video-anc.h \\
/drm/include/gstreamer-1.0/gst/video/video-blend.h \\
/drm/include/gstreamer-1.0/gst/video/video-chroma.h \\
/drm/include/gstreamer-1.0/gst/video/video-color.h \\
/drm/include/gstreamer-1.0/gst/video/video-converter.h \\
/drm/include/gstreamer-1.0/gst/video/video-dither.h \\
/drm/include/gstreamer-1.0/gst/video/video-enumtypes.h \\
/drm/include/gstreamer-1.0/gst/video/video-event.h \\
/drm/include/gstreamer-1.0/gst/video/video-format.h \\
/drm/include/gstreamer-1.0/gst/video/video-frame.h \\
/drm/include/gstreamer-1.0/gst/video/video-info.h \\
/drm/include/gstreamer-1.0/gst/video/video-multiview.h \\
/drm/include/gstreamer-1.0/gst/video/video-overlay-composition.h \\
/drm/include/gstreamer-1.0/gst/video/video-prelude.h \\
/drm/include/gstreamer-1.0/gst/video/video-resampler.h \\
/drm/include/gstreamer-1.0/gst/video/video-scaler.h \\
/drm/include/gstreamer-1.0/gst/video/video-tile.h \\
/drm/include/gstreamer-1.0/gst/video/videodirection.h \\
/drm/include/gstreamer-1.0/gst/video/videoorientation.h \\
/drm/include/gstreamer-1.0/gst/video/videooverlay.h \\
/drm/lib/libgstallocators-1.0.so \\
/drm/lib/libgstapp-1.0.so \\
/drm/lib/libgstaudio-1.0.so \\
/drm/lib/libgstfft-1.0.so \\
EOF
test $GL && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/libgstgl-1.0.so \\
EOF
cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/libgstpbutils-1.0.so \\
/drm/lib/libgstriff-1.0.so \\
/drm/lib/libgstrtp-1.0.so \\
/drm/lib/libgstrtsp-1.0.so \\
/drm/lib/libgstsdp-1.0.so \\
/drm/lib/libgsttag-1.0.so \\
/drm/lib/libgstvideo-1.0.so \\
/drm/lib/pkgconfig/gstreamer-allocators-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-app-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-audio-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-fft-1.0.pc \\
EOF
test $GL && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/pkgconfig/gstreamer-gl-1.0.pc \\
EOF
cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/pkgconfig/gstreamer-pbutils-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-plugins-base-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-riff-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-rtp-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-rtsp-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-sdp-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-tag-1.0.pc \\
/drm/lib/pkgconfig/gstreamer-video-1.0.pc

exec=\\
/drm/bin/gst-device-monitor-1.0 \\
/drm/bin/gst-discoverer-1.0 \\
/drm/bin/gst-play-1.0
/drm/lib/libgstallocators-1.0.so.0 \\
/drm/lib/libgstapp-1.0.so.0 \\
/drm/lib/libgstaudio-1.0.so.0 \\
/drm/lib/libgstfft-1.0.so.0 \\
EOF
test $GL && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/libgstgl-1.0.so.0 \\
EOF
cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/libgstpbutils-1.0.so.0 \\
/drm/lib/libgstriff-1.0.so.0 \\
/drm/lib/libgstrtp-1.0.so.0 \\
/drm/lib/libgstrtsp-1.0.so.0 \\
/drm/lib/libgstsdp-1.0.so.0 \\
/drm/lib/libgsttag-1.0.so.0 \\
/drm/lib/libgstvideo-1.0.so.0 \\
/drm/lib/gstreamer-1.0/libgstadder.so \\
EOF
test $ALSA && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/gstreamer-1.0/libgstalsa.so \\
EOF
cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/gstreamer-1.0/libgstapp.so \\
/drm/lib/gstreamer-1.0/libgstaudioconvert.so \\
/drm/lib/gstreamer-1.0/libgstaudiomixer.so \\
/drm/lib/gstreamer-1.0/libgstaudiorate.so \\
/drm/lib/gstreamer-1.0/libgstaudioresample.so \\
/drm/lib/gstreamer-1.0/libgstaudiotestsrc.so \\
/drm/lib/gstreamer-1.0/libgstcompositor.so \\
/drm/lib/gstreamer-1.0/libgstencoding.so \\
/drm/lib/gstreamer-1.0/libgstgio.so \\
EOF
test $TREMOR && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/gstreamer-1.0/libgstivorbisdec.so \\
EOF
cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/gstreamer-1.0/libgstogg.so \\
EOF
test $GL && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/gstreamer-1.0/libgstopengl.so \\
EOF
test $OPUS && cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/gstreamer-1.0/libgstopus.so \\
EOF
cat >> $DESTDIR/drm/share/pkgconfig/gst-plugins-base-build.pc << EOF
/drm/lib/gstreamer-1.0/libgstpango.so \\
/drm/lib/gstreamer-1.0/libgstpbtypes.so \\
/drm/lib/gstreamer-1.0/libgstplayback.so \\
/drm/lib/gstreamer-1.0/libgstoverlaycomposition.so \\
/drm/lib/gstreamer-1.0/libgstrawparse.so \\
/drm/lib/gstreamer-1.0/libgstsubparse.so \\
/drm/lib/gstreamer-1.0/libgsttcp.so \\
/drm/lib/gstreamer-1.0/libgsttypefindfunctions.so \\
/drm/lib/gstreamer-1.0/libgstvideoconvert.so \\
/drm/lib/gstreamer-1.0/libgstvideorate.so \\
/drm/lib/gstreamer-1.0/libgstvideoscale.so \\
/drm/lib/gstreamer-1.0/libgstvideotestsrc.so \\
/drm/lib/gstreamer-1.0/libgstvolume.so \\
/drm/lib/gstreamer-1.0/libgstvorbis.so
EOF

exit

# patch
--- gst-plugins-base-1.16.3.orig/configure.ac
+++ gst-plugins-base-1.16.3/configure.ac
@@ -25,7 +25,7 @@
 AC_CONFIG_HEADERS([config.h])
 
 dnl AM_MAINTAINER_MODE only provides the option to configure to enable it
-AM_MAINTAINER_MODE([enable])
+AM_MAINTAINER_MODE
 
 dnl sets host_* variables
 AC_CANONICAL_HOST
--- gst-plugins-base-1.16.3.orig/gst-libs/gst/fft/fft.h
+++ gst-plugins-base-1.16.3/gst-libs/gst/fft/fft.h
@@ -22,7 +22,6 @@
 #ifndef __GST_FFT__H__
 #define __GST_FFT__H__
 
-#include <gst/fft/gstfft.h>
 #include <gst/fft/gstffts16.h>
 #include <gst/fft/gstffts32.h>
 #include <gst/fft/gstfftf32.h>
--- gst-plugins-base-1.16.3.orig/gst-libs/gst/gl/Makefile.am
+++ gst-plugins-base-1.16.3/gst-libs/gst/gl/Makefile.am
@@ -150,9 +150,6 @@
 gstgl_gir_sources += egl/gstgldisplay_egl.c
 endif
 
-configexecincludedir = $(libdir)/gstreamer-@GST_API_VERSION@/include/gst/gl
-nodist_configexecinclude_HEADERS = $(built_sys_header_configure)
-
 libgstgl_@GST_API_VERSION@_la_CFLAGS = \
 	-DBUILDING_GST_GL \
 	$(GST_PLUGINS_BASE_CFLAGS) \
@@ -181,7 +178,7 @@
 
 CLEANFILES = $(BUILT_SOURCES)
 
-nodist_libgstgl_@GST_API_VERSION@include_HEADERS = $(built_headers)
+nodist_libgstgl_@GST_API_VERSION@include_HEADERS = $(built_headers) $(built_sys_header_configure)
 nodist_libgstgl_@GST_API_VERSION@_la_SOURCES = $(BUILT_SOURCES)
 
 include $(top_srcdir)/common/gst-glib-gen.mak
--- gst-plugins-base-1.16.3.orig/gst-libs/gst/gl/egl/Makefile.am
+++ gst-plugins-base-1.16.3/gst-libs/gst/gl/egl/Makefile.am
@@ -24,6 +24,7 @@
 	-I$(top_srcdir)/gst-libs \
 	-I$(top_builddir)/gst-libs \
 	-DBUILDING_GST_GL \
+	-DEGL_NO_PLATFORM_SPECIFIC_TYPES \
 	$(GL_CFLAGS) \
 	$(GST_PLUGINS_BASE_CFLAGS) \
 	$(GST_BASE_CFLAGS) \
--- gst-plugins-base-1.16.3.orig/gst-libs/gst/riff/riff.h
+++ gst-plugins-base-1.16.3/gst-libs/gst/riff/riff.h
@@ -23,7 +23,6 @@
 #define __GST_RIFF_H__
 
 #include <gst/riff/riff-prelude.h>
-#include <gst/riff/riff-ids.h>
 #include <gst/riff/riff-media.h>
 #include <gst/riff/riff-read.h>
 
--- gst-plugins-base-1.16.3.orig/gst-libs/gst/rtsp/rtsp.h
+++ gst-plugins-base-1.16.3/gst-libs/gst/rtsp/rtsp.h
@@ -24,7 +24,6 @@
 
 #include <gst/rtsp/rtsp-prelude.h>
 
-#include <gst/rtsp/gstrtsp.h>
 #include <gst/rtsp/gstrtspconnection.h>
 #include <gst/rtsp/gstrtspdefs.h>
 #include <gst/rtsp/gstrtsp-enumtypes.h>
--- gst-plugins-base-1.16.3.orig/gst-libs/gst/sdp/gstsdpmessage.h
+++ gst-plugins-base-1.16.3/gst-libs/gst/sdp/gstsdpmessage.h
@@ -47,7 +47,7 @@
 
 #include <glib.h>
 #include <gst/gst.h>
-#include <gst/sdp/sdp.h>
+#include <gst/sdp/gstsdp.h>
 
 G_BEGIN_DECLS
 
--- gst-plugins-base-1.16.3.orig/gst-libs/gst/sdp/sdp.h
+++ gst-plugins-base-1.16.3/gst-libs/gst/sdp/sdp.h
@@ -24,7 +24,6 @@
 
 #include <gst/sdp/sdp-prelude.h>
 
-#include <gst/sdp/gstsdp.h>
 #include <gst/sdp/gstsdpmessage.h>
 #include <gst/sdp/gstmikey.h>
 
--- gst-plugins-base-1.16.3.orig/pkgconfig/gstreamer-gl.pc.in
+++ gst-plugins-base-1.16.3/pkgconfig/gstreamer-gl.pc.in
@@ -13,4 +13,4 @@
 Requires: gstreamer-video-@GST_API_VERSION@ gstreamer-base-@GST_API_VERSION@ gstreamer-@GST_API_VERSION@
 
 Libs: -L${libdir} -lgstgl-@GST_API_VERSION@
-Cflags: -I${includedir} -I${libdir}/gstreamer-@GST_API_VERSION@/include @GL_CFLAGS@
+Cflags: -I${includedir} @GL_CFLAGS@
--- gst-plugins-base-1.16.3.orig/tests/examples/fft/fftrange.c
+++ gst-plugins-base-1.16.3/tests/examples/fft/fftrange.c
@@ -18,10 +18,7 @@
  */
 
 #include <stdio.h>
-#include <gst/fft/gstffts16.h>
-#include <gst/fft/gstffts32.h>
-#include <gst/fft/gstfftf32.h>
-#include <gst/fft/gstfftf64.h>
+#include <gst/fft/fft.h>
 
 /* effectively max range seems to be 1/4 of what it should be */
 
# source
https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.16.3.tar.xz
